### ISSUE TITLE
ts-styled-plugin doesn't recognise adjacent variables in padding shorthand value

### ISSUE THREAD

**Issue by riglar** [Microsoft/typescript-styled-plugin#62]
IDE: VS Code 1.24.1
typescript-styled-plugin@0.6.3
Implemented via tsconfig.json plugins

```
const Wrapper = styled.div`
  padding: ${spaces.s} ${spaces.l};
`
```

**Expected behaviour**
Example about does not warn

**Actual behaviour**
```
warning: '[ts-styled-plugin] semi-colon expected'
```

The plugin is highlighting the `s` of  `spaces.l` which seems to me that it does it doesn't recognise shorthand being applied in this way. Note that the below works without issue:
```
const Wrapper = styled.div`
  padding: 4px 10px;
`
```

**Comment by cgatno** [comment]
Just wanted to note that `padding: ${spacing} 0;` also works without issue, so the problem is limited to the case in which there are two adjacent expressions.

**Comment by ppozniak** [comment]
Same issue here.
`margin: ${rem('36px')} ${rem('50px')} ${rem('50px')} ${rem('50px')};` such chain will always higlight the last expression with **semi-colon expected**. In this case in 58th column so that is between "{" and "r"

**Comment by mvasin** [comment]
And here's my case:

<img width="490" alt="screen shot 2018-08-16 at 10 12 43" src="https://user-images.githubusercontent.com/12434833/44193933-32abd880-a13d-11e8-8b08-a0a028d49d28.png">

**Comment by DJTB** [comment]
Similar case:
```
&:nth-last-child(n+2) {
    ${AvatarWrapper}:after {
       /* normal css */
    }
}
```
(which was actually just an attempt to work-around `} expected`) for:
```
&:nth-last-child(n+2) ${AvatarWrapper}:after {
   /* normal css */
}
```
